### DISCUSSION TITLE
Dynamically path by selecting a parameter in variables

### DISCUSSION THREAD

**Discussion by anatoly-kor** [grafana/grafana-json-datasource#102] [Q&A] [answered]
I am trying to make a request for dynamically path by selecting a parameter in variables. 

I want choose in variables:
![image](https://user-images.githubusercontent.com/30874961/115963512-4a5fcf80-a528-11eb-9c0d-654202084364.png)

and make a request for example:

`/v1/api/TEST/` or `/v1/api/EXAMPLE/` etc

Is it possible?
How I can make it?

Thanks for your help

**Comment by marcusolsson** [comment, marked as answer]
Have you tried using variables in the Path field?

<img width="264" alt="Screenshot 2021-04-26 at 10 10 27@2x" src="https://user-images.githubusercontent.com/8396880/116050339-a6853980-a677-11eb-970d-b9c72cb87b47.png">

**Reply by anatoly-kor** [reply]
Thank you very much!

**Comment by charishma1** [comment]
Hi,I am trying to fetch json data from dynmaic urls like above.
But am able to fetch the data in multiple panels for each value of the variable used in the url when i enable panel repetition option.
Is there a way that i can get the data off all the dynamic urls in the same panel ?
